game/src/game/game_player/effect_sound_player.rs

```diff
@@ -87,4 +147,19 @@ impl EffectSoundPlayer {
             }
         }
     }
+
+    pub fn play_combo_sound(&mut self, game_result: &GameResult, audio_manager: &mut AudioManager) {
+        if game_result.combo > 0 && (game_result.combo % 10 == 0) {
+            if !self.combo_sound_played {
+                if let Some(combo_sound) = self.combo_sounds.choose(&mut rand::thread_rng()) {
+                    audio_manager
+                        .play(combo_sound.clone())
```

Comment:
Volume can set here by using `combo_sound.clone().with_settings(StaticSoundSettings::default().volume(50.0))` instead of `combo_sound.clone()`